test/performance/idpay/getWalletDetails.js

```diff
@@ -16,40 +16,44 @@ import {
     logErrorResult,
 } from '../../common/dynamicScenarios/utils.js'
 
+// Environments allowed to be tested
 const REGISTERED_ENVS = [DEV, UAT]
 const baseUrl = getBaseUrl(REGISTERED_ENVS, 'io')
 
 const application = 'idpay'
 const testName = 'getWalletDetail'
 
 // Set up data for processing, share data among VUs
-const cfPanList = new SharedArray('cfPanList', getFCPanList)
+const cfList = new SharedArray('cfList', getFCList)
 
-// K6 configurations
-export const options = defaultApiOptionsBuilder(application, testName)
+// Dynamic scenarios' K6 configuration
+export const options = defaultApiOptionsBuilder(
+    application,
+    testName,
+    [WALLET_API_NAMES.getWalletDetail] // applying apiName tags to thresholds
+)
 
+// K6 summary configuration
 export const handleSummary = defaultHandleSummaryBuilder(application, testName)
 
 export default () => {
-    const scenarioEntity = getScenarioTestEntity(cfPanList)
-    const cf = scenarioEntity.cf
+    const cf = getScenarioTestEntity(cfList).FC
     const params = { headers: buildIOAuthorizationHeader(cf) }
 
-    group('Payment Instrument API', () => {
-        group('Should enroll pgpan', () => {
-            const res = getWalletDetail(
-                baseUrl,
-                IDPAY_CONFIG.CONTEXT_DATA.initiativeId,
-                params
-            )
-
-            if (res.status != 200) {
-                logErrorResult('Enrollment Carte', res, true)
-                return
-            }
-
-            assert(res, [statusOk()])
-        })
+
+    group('When onboarding is completed, get wallet detail', () => {
+        const res = getWalletDetail(
+            baseUrl,
+            IDPAY_CONFIG.CONTEXT_DATA.initiativeId,
+            params
+        )
+
+        if (res.status != 200) {
+            logErrorResult(`Wallet associated to user with cf [${cf}] not found`, res, true)
+            return
+        }
+
+        assert(res, [statusOk()])
```

Comment:
switch these instructions!
if result !== 200 you are exiting, without performing the below assert